bitcoin-magazine/bitcoin-2024/making-bitcoin-more-private-with-cisa.md

```diff
@@ -184,110 +187,119 @@ So that's how I would encourage you to think about it from the start.
 
-Speaker 1: 00:19:22
+Nifi: 00:19:22
 
-The final form of SIGAG as we go all the way to the block level.
+The final form of sigag as we go all the way to the block level.
 Cool, so we only have a few minutes left.
 Maybe we can spend a little bit of time talking about, okay, so we almost had it, we didn't get it.
-What would it take, what's left from here to get it to the point where we could soft fork it in?
+What's left from here to get it to the point where we could soft-fork it in?
 
-Speaker 0: 00:19:45
+Fabian Jahr: 00:19:45
 
 There's still quite a few things to hammer out in terms of all of the details, like defining a spec.
 
 ## Signature Aggregation and Getting Involved
 
-Speaker 0: 00:19:53
-
-So for the half-arc, there's a BIP draft, but it's not a pull request on BIP repository, but there's just minor to-dos like test vectors there that are open however for the full aggregation we still need to develop the signature scheme, so that requires quite a bit of effort from people reversed in cryptography and specifically on the side of the interactivity and We will also want to have a security proof for that scheme that comes out on the other hand and that means people with Very specific talents and an experience in that area will also be needed in process so so these I think are the big to-dos that we have and Yeah The people that can do this stuff need to come together and work on it.
+So for the half-arc, there's a BIP draft, but it's not a pull request on BIP repository, but there's just minor to-dos like test vectors there that are open however for the full aggregation we still need to develop the signature scheme, so that requires quite a bit of effort from people reversed in cryptography and specifically on the side of the interactivity.
+We will also want to have a security proof for that scheme that comes out on the other hand and that means people with very specific talents and an experience in that area will also be needed in process.
+These I think are the big to-dos that we have.
+The people that can do this stuff need to come together and work on it.
 Personally, I spend quite a bit of time on it right now.
 But maybe something to everyone watching this or in the audience, if you want to get involved, let us know.
 
-Speaker 3: 00:21:01
+Jameson Lopp: 00:21:01
 
 Call your local developer and demand signature aggregation today.
 
-Speaker 2: 00:21:05
+Craig Raw: 00:21:05
 
 Yeah, and just to that point, Fabian has developed a great site where you can learn more.
-So, you know, if the concepts that we've talked about here today make sense, if you want to understand them better.
-If you go to sizeresearch.org, that's C-I-S-A research dot org, you know you can really learn much more about it.
+So, if the concepts that we've talked about here today make sense, if you want to understand them better.
+If you go to sizeresearch.org, that's C-I-S-A research dot org, you can really learn much more about it.
```

Comment:
The domain was picked wrong by the AI transcription, but then it spells it letter by letter. This is something that we expect the reviewers to fix and honestly @markon1-a this is a signal that you are still not paying attention to the full context of the transcript.
```suggestion
If you go to <cisaresearch.org> you can really learn much more about it.
```

Reply:
Yes this was an obvious one I didn't concentrate on.